### PR TITLE
Document Explicit/Implicit operators for System.Numerics.Complex.

### PR DESCRIPTION
**Pull request by tompratt-AQ**
Document all Explicit and Implicit operators for System.Numerics.Complex.

## Summary
Pulled information from MSDN to document 2 Explicit and 10 Implicit operators for System.Numerics.Complex.

## Suggested Reviewers
@rpetrusha

### CODE REVIEW

File: xml/System.Numerics/Complex.xml [hunk 1]
```diff
@@ -1806,10 +1942,24 @@
...
+        <returns>An object that contains the value of the <paramref name="value" /> parameter as its real part and zero as its imaginary part.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The overloads of the <xref:System.Numerics.Complex.op_Implicit%2A> operator define the types from which a compiler can automatically convert a <xref:System.Numerics.Complex> object without an explicit casting operator (in C#) or a call to a conversion function (in Visual Basic). They are widening conversions that do not involve data loss and do not throw an <xref:System.OverflowException>.
+
+ This overload lets the compiler handle conversions from an unsigned 32-bit integer to a complex number, as the following example shows. Note that the result of the conversion is a complex number whose real part is equal to the unsigned 32-bit integer and whose imaginary part is equal to zero.
+
+ [!code-csharp[System.Numerics.Complex.Implicit#9](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.numerics.complex.implicit/cs/implicit1.cs#9)]
```
Comment (rpetrusha) [inline xml/System.Numerics/Complex.xml]: The C# code is incorrect. Could you change it, Tom? In its present form:
```
private static void ComplexFromUInt32()
   {
      // <Snippet9>
      int intValue = 197461;
      System.Numerics.Complex c1 = intValue;
      Console.WriteLine(c1);
      // The example displays the following output:
      //       (197461, 0)
      // </Snippet9>
   }
```
Change
``` 
int intValue = 197461;
```
to
```
int intValue = 197461;
```
The VB code is OK.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
This looks really good, @tompratt-AQ. Just one thing to change -- when I wrote the original code examples, I used an `int` instead of a `uint` in one of the C# code samples.

**Comment by tompratt-AQ** [comment]
@rpetrusha  Thanks Ron. I changed int intValue; to uint value; and System.Numerics.Complex c1 = intValue; to System.Numerics.Complex c1 = value; as you suggested in your email.

**Comment by rpetrusha** [comment]
Thanks, @tompratt-AQ. I'll review it first thing tomorrow.

**Comment by rpetrusha** [comment]
@tompratt-AQ, this looks good. I'll merge now.

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**